Rename fiscal calendar options (#58)
tableau/hyper-db: website/docs/releases.md

### Upcoming Release

* .Net Deprecation: As announced last release, we are deprecating the .NET version of Hyper API. We decided to [open-source its source code](https://github.com/tableau/hyper-api-dotnet), so whoever might be interested in maintaining it can pick up where we left.

> * Some of the [Fiscal Calendar Options](/docs/sql/scalar_func/datetime#fiscal-calendar-options) were renamed:
>   * `start_month` was renamed to `fiscal_year_start_month`.
>   * `first_day_of_week` was renamed to `first_day_of_fiscal_week`.

Reviewer: ```suggestion
* Some of the [Fiscal Calendar Options](/docs/sql/scalar_func/datetime#fiscal-calendar-options) are renamed:
  * `start_month` is to `fiscal_year_start_month`.
  * `first_day_of_week` is renamed to `first_day_of_fiscal_week`.
```
